docs/application/dotnet/guides/text-input/input-method.md

```diff
@@ -86,6 +86,41 @@ To start the IME application life-cycle:
     }
     ```
 
+## Making a Floating Keyboard
+
+To make the floating keybaord to allow the user to move the keyboard:
```

Comment:
Please change to:
To use the floating keyboard to allow the user to move the keyboard:

Please can you verify whether the sentence is technically correct?